charmstore_test.go

```diff
@@ -496,7 +496,7 @@ func (s *charmStoreRepoSuite) TestLatest(c *gc.C) {
 	// Run the tests.
 	for i, test := range tests {
 		c.Logf("test %d: %s", i, test.about)
-		revs, err := s.repo.Latest(test.urls...)
+		revs, err := s.repo.(*charmrepo.CharmStore).Latest(test.urls...)
```

Comment:
Ah, interesting. Probably NewCharmStore should return *CharmStore not interface then. That would be conventional.